django-web-app/cogniguard/urls.py

```diff
@@ -1,40 +1,23 @@
 """
 URL configuration for cogniguard project.
-
 """
 from django.contrib import admin
 from django.urls import path,include
 from django.views.generic import RedirectView
 from django.conf import settings
 from django.conf.urls.static import static
-
-from django.contrib.auth import views as auth_views
-
 # Admin panel customization
 admin.site.site_header = "CogniGuard Admin"
 admin.site.site_title = "CogniGuard Admin Portal"
-
-
 urlpatterns = [
     path('admin/', admin.site.urls),
     # path("logout/", RedirectView.as_view(url="/admin/logout/")),
     path("", include("home.urls")),
-
-    # for rendering reset password cnf template
```

Comment:
same issue, 
you've removed some paths required for password reset which have been added in recent commit. 
fix it.